package.json

```diff
@@ -24,9 +24,10 @@
   "pre-commit": "lint:staged",
   "license": "MIT",
   "engines": {
-    "npm": "7.x",
-    "node": "^v14.17.0 || >=v16.0.0"
+    "node": "^v14.17.0 || >=v16.0.0",
+    "yarn": "3.0.2"
```

Comment:
If everyone is using Corepack, everyone will be on the same version (it’s like nvm but for package managers). We could still loosen the engine field, but maybe we shouldn’t encourage devs to use something other than Corepack?

Reply:
As long as `^v14.17.0` is in there corepack won't be available for those people, right? In that case a more liberal `yarn` version would be preferable.

Reply:
That's not correct, folks can install Corepack using `npm i -g corepack` which works on Node.js v12+ (it also works on v10.x but the support is not official).

Reply:
I'm just thinking if people want to contribute and have yarn >=3x installed but not corepack, that should be allowed. Corepack makes it seamless but I'm not sure if we need to force it on people?

Reply:
I'm not sure, Yarn themsleves do not recommend this approach: their official guide says users should not install Yarn v2+ globally but Yarn v1, so that wouldn't work for us (source: https://yarnpkg.com/getting-started/install).
IMHO we shouldn't support users using Yarn outside of Corepack (they can still use CLI arguments to use it with a different version).

Reply:
Good point on yarn no longer being recommended to install globally. I haven't used it since V1. We can keep this strict then.